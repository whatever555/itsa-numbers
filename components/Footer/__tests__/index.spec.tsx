import React from 'react'

import { render } from '@testing-library/react'

import { Footer } from '../'

describe('Footer', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Footer />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
