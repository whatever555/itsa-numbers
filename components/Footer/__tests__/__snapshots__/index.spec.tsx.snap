// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="sc-bdfBwQ bhFJQJ"
  >
    Calculator 2021
  </footer>
</DocumentFragment>
`;
